public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -195,27 +193,25 @@ export class SharedPreferences extends PureComponent<Props, State> {
                 />
               </Field>
 
-              {i18nFlag ? (
-                <Field
-                  label={
-                    <Label htmlFor="locale-select">
-                      <span className={styles.labelText}>
-                        <Trans i18nKey="shared-preferences.fields.locale-label">Language</Trans>
-                      </span>
-                      <FeatureBadge featureState={FeatureState.beta} />
-                    </Label>
-                  }
-                  data-testid="User preferences language drop down"
-                >
-                  <Select
-                    value={languages.find((lang) => lang.value === language)}
-                    onChange={(lang: SelectableValue<string>) => this.onLanguageChanged(lang.value ?? '')}
-                    options={languages}
-                    placeholder={t('shared-preferences.fields.locale-placeholder', 'Choose language')}
-                    inputId="locale-select"
-                  />
-                </Field>
-              ) : null}
+              <Field
+                label={
+                  <Label htmlFor="locale-select">
+                    <span className={styles.labelText}>
+                      <Trans i18nKey="shared-preferences.fields.locale-label">Language</Trans>
+                    </span>
+                    <FeatureBadge featureState={FeatureState.beta} />
```

Comment:
If we do this, then we should remove the badge

Reply:
@joshhunt i've removed it... but now i wonder if we should just leave it anyway? 🤔 it gives some indication that the feature is still WIP, which is kinda true tbh 😬

Reply:
Well, this ties into my uncertainly about all of this - _because_ we have such poor coverage of translations and it's still 'WIP' then maybe we should keep the feature flag there?